packages/mdc-menu/adapter.ts

```diff
@@ -65,6 +65,11 @@ export interface MDCMenuAdapter {
    */
   getElementIndex(element: Element): number;
 
+  /**
+   * @return List item element.
+   * @param index Index of list item within menu.
+   */
+  getListItemAtIndex(index: number): Element | null;
```

Comment:
Foundation will always send valid index number to adapter that makes return type non-nullable.

Reply:
What should i have for the defaultFoundation value then? I could also switch this to undefined.

Reply:
`null as Element`? This keeps the TSLint happy. We don't want to create a dummy element just to provide value to default adapter.

WDYT?

Reply:
ya I don't think we should create a dummy element. However isn't `Element | null` technically correct, since at one point it is `null`?